onepassword/util/date.go

```diff
@@ -0,0 +1,25 @@
+package util
```

Comment:
The existing `utils.go` is a part of `cli` package that contains the function more specific for the CLI, so I'd not put the new functions there because they are not related to the CLI.

Reply:
That makes sense. I totally missed that the file is part of the `cli` package. Then this code can stay here. 😄